packages/contracts/test/diamond/DiamondTestSetup.sol

```diff
@@ -436,7 +483,7 @@ abstract contract DiamondSetup is DiamondTestHelper {
             )
         });
 
-        FacetCut[] memory cuts = new FacetCut[](15);
+        FacetCut[] memory cuts = new FacetCut[](16);
```

Comment:
Also here, related to my previous comment above.

Reply:
Could be updated with new branches

Reply:
My fear is when two branches are both looking to merge in a new facet and both have incremented the facet count by one. The first one will pass the merge CI. The second one will fail. It is a simple problem but technically it would be a perpetual problem. That's why I'm bringing this to the team's attention to see if there's a better way to do this.

Reply:
@zgorizzo69